OPEN_SOURCE_POLICY.md

```diff
@@ -0,0 +1,163 @@
+# Open Source Policy
+
+## 1. Introduction
+
+This Open Source Policy outlines the licensing, contribution, and compliance
+requirements for all code released under the Elixir project. By adhering to
+these guidelines, we ensure that our community, maintainers, and contributors
+uphold both legal and ethical standards while fostering a collaborative,
+transparent environment.
+
+This policy exists to support and protect the Elixir community. It aims to
+balance openness, collaboration, and respect for all contributors’ rights,
+ensuring that Elixir remains a trusted and innovative open source project.
+
+## 2. Scope
+
+This policy applies to the Elixir Programming language, located at
+https://github.com/elixir-lang/elixir. It covers every file, and contribution
+made, including documentation and any associated assets.
+
+## 3. Licensing
+
+All code released by the Elixir team is licensed under the
+[Apache-2.0](./LICENSES/Apache-2.0.txt) license. Additionally, the following
+licenses are recognized as permissible in this project:
+
+- The Unicode license, as documented at
+  [LicenseRef-scancode-unicode](./LICENSES/LicenseRef-scancode-unicode.txt)
+- The Elixir Trademark Policy, as documented at
```

Comment:
Is this a license? Are any files in our repository licensed under this?

Reply:
@ericmj It was registered as a license in the AboutCode LicenseDB https://github.com/aboutcode-org/scancode-toolkit/issues/4131 and added as a license: https://github.com/elixir-lang/elixir/blob/main/LICENSES/LicenseRef-elixir-trademark-policy.txt

This is also conforming to SPDX since custom licences can be created if they start with `LicenseRef-`.

We are specifically covering the logo image, which is part of the documentation.